avocado_virt/defaults.py

```diff
@@ -30,6 +30,11 @@
         qemu_bin = 'qemu'
 
 try:
+    no_kvm = settings.get_value('virt.qemu', 'no_kvm')
+except SettingsError:
+    no_kvm = False
```

Comment:
One more thing regarding the high level overview, how about using `kvm` and use the values `on`/`off` and when not defined check the `/dev/kvm`? In the future we'd be able to expand it handle the `kvm-pr`, `kvm-hv` and other variants.